generated/niswitch/enums.py

```diff
@@ -226,52 +226,43 @@ class RelayPosition(Enum):
 class ScanAdvancedOutput(Enum):
     NONE = 0
     '''
-    The switch module does not produce a Scan Advanced Output trigger.
+    The switch device does not produce a Scan Advanced Output trigger.
     '''
     EXTERNAL = 2
     '''
-    The switch module produces the Scan Advanced Output trigger on the
-    external trigger output.
+    External Trigger. The switch device produces the Scan Advanced Output  trigger on the external trigger output.
     '''
-    PXI_TRIG0 = 111
+    TTL0 = 111
```

Comment:
On DMM it went TTL -> PXI_TRIG and on NI-SWITCH it went PXI_TRIG -> TTL??
What is wrong with us!?

Reply:
I'm trying to dig up NI internal bug requests but the DB is down!

Reply:
Do we want Python to try to be consistent within the Python bindings or for each API to be consistent with the C API of that driver? I can see pros and cons for each.

Reply:
I think we need to fix the C APIs. It’s embarrasing!
😳